include/functions.h

```diff
@@ -2555,7 +2556,7 @@ s32 Entrance_GetTransitionFlags(u16 entranceIndex);
 // void func_80132E9C(void);
 // void func_80132FDC(void);
 // void func_80133000(void);
-// void func_80133038(void);
+s32 func_80133038(GlobalContext* globalCtx, UNK_PTR arg1, UNK_PTR arg2);
```

Comment:
```suggestion
s32 func_80133038(GlobalContext* globalCtx, UNK_PTR arg1, struct_80133038_arg2* arg2);
```
Since a struct was made for this argument, it shouldn't be left as unk as it can confuse others encountering this function, not knowing that a struct exists for it, and making their own when it should be the same type.

Reply:
Ah good catch.